foyer/forcefield.py

```diff
@@ -436,26 +444,25 @@ def createSystem(self, topology, atomtype=True, use_residue_map=True,
             else:
                 find_atomtypes(topology, forcefield=self)
 
-        data = app.ForceField._SystemData()
-        data.atoms = list(topology.atoms())
-        for atom in data.atoms:
-            data.excludeAtomWith.append([])
+        self._SystemData.atoms = list(topology.atoms())
```

Comment:
Could you limit this to just adding a single line here which sets `data = self._SystemData()` in place of the existing `data = app.ForceField._SystemData()`?